packages/rich-text/src/component/index.js

```diff
@@ -140,7 +144,7 @@ export function useRichText( {
 		if ( disableFormats ) {
 			_value.current = newRecord.text;
 		} else {
-			_value.current = toHTMLString( {
+			_value.current = new RichTextValue( {
```

Comment:
This will be a breaking change for `typeof value === 'string'`, won't it? I guess it's okay if it's never documented 😅  (however it is documented in the `RichText` component in the block-editor package).